src/Sulu/Bundle/DistributionBundle/Tests/Unit/Composer/ScriptHandlerTest.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\DistributionBundle\Tests\Unit\Composer;
+
+use Composer\Composer;
+use Composer\IO\IOInterface;
+use Composer\Package\RootPackageInterface;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+use Sulu\Bundle\DistributionBundle\Composer\ScriptHandler;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ScriptHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    private $tempDir;
+
+    private $resourceDir;
+
+    private $previousWd;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Composer
+     */
+    private $composer;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|IOInterface
+     */
+    private $io;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|RootPackageInterface
+     */
+    private $rootPackage;
+
+    /**
+     * @var Event
+     */
+    private $event;
+
+    protected function setUp()
+    {
+        $this->tempDir = self::makeTempDir(__CLASS__);
+        $this->previousWd = getcwd();
+        $this->composer = $this->getMockBuilder('Composer\Composer')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->io = $this->getMock('Composer\IO\IOInterface');
+        $this->rootPackage = $this->getMock('Composer\Package\RootPackageInterface');
+        $this->event = new Event(ScriptEvents::POST_INSTALL_CMD, $this->composer, $this->io);
+
+        $this->composer->expects($this->any())
+            ->method('getPackage')
+            ->willReturn($this->rootPackage);
```

Comment:
We are using Prophecy for mocks, because we like the syntax better. Any reason you are using the PHPUnit builtins here?